custom/css/style.css

```diff
@@ -1552,3 +1552,23 @@ ul.faq-answer li a:hover {
     min-height: 50px;
     padding: 1em;
 }
+
+
+#bs-example-navbar-collapse-1 > ul > li.dropdown.open > ul > li:nth-child(2) > a.dropdown-toggle{
+  display: none;}
+
+#bs-example-navbar-collapse-1 > ul > li.dropdown.open > ul{
+  display: block;
+  width: 100%;
+}
+
+#page-top > nav > div > a > img{
+  height: 60px !important;
+}
+
+.navbar-custom ul.nav ul.dropdown-menu li a{
+  white-space: normal;
+}
+.navbar-custom ul.nav ul.dropdown-menu li a {
```

Comment:
I think it will look better without center aligned since other nav items are left aligned. Just remove this style.